scss/theme/_home.scss

```diff
@@ -312,6 +312,7 @@
 			display: inline-block;
 			border-radius: 25px;
 			font-weight:400;
+			transition: background 0.5s, border 0.5s, color 0.5s;
```

Comment:
This one ^^

Reply:
Yo awesome :) really appreciate this man!